src/main/java/data/workouts/WorkoutList.java

```diff
@@ -0,0 +1,44 @@
+package data.workouts;
+
+import data.exercises.ExerciseList;
+import data.exercises.InvalidExerciseException;
+
+import java.util.ArrayList;
+
+public class WorkoutList {
+    ExerciseList exerciseList;
+    ArrayList<Workout> workoutsList = new ArrayList<>();
+
+    public WorkoutList(ExerciseList exerciseList) {
+        this.exerciseList = exerciseList;
+    }
+
+    public Workout createWorkout(String userArgument) throws ArrayIndexOutOfBoundsException,
```

Comment:
Just a suggestion, if it would be better to rename createWorkout to createAndAddWorkout. Otherwise, it's good :)!

Reply:
Fixed in a61df74